package.json

```diff
@@ -51,7 +51,7 @@
     "uuid": "^9.0.0"
   },
   "peerDependencies": {
-    "@gravity-ui/nodekit": "^1.4.0"
+    "@gravity-ui/nodekit": "file:../nodekit"
```

Comment:
Don't forget to clean it

Reply:
Yep, sure, this is for show potential changes with new nodekit